### PR TITLE
Add support for nodeSelector and tolerations

### PR DESCRIPTION
**Pull request by athak**
This PR adds support for adding `nodeSelector` and `tolerations` to the AWX pods.

### CODE REVIEW

File: roles/installer/defaults/main.yml [hunk 1]
```diff
@@ -38,6 +38,22 @@ tower_route_host: ''
...
+tower_node_selector: ''
+
+# Add node tolerations for the AWX pods. Specify as literal block. E.g.:
+# tower_tolerations: |
+#   - key: "dedicated"
+#     operator: "Equal"
+#     value: "AWX"
+#     effect: "NoSchedule"
+tower_tolerations: ''
+
```
Comment (tchellomello) [inline roles/installer/defaults/main.yml:56]: @athak would you be willing to add this section also on the docs at https://github.com/ansible/awx-operator#advanced-configuration? I think that will be important for our users to have a reference point on how to configure it without looking at the code.

**Comment by Spredzy** [comment]
@athak looks great. Thanks for the PR. In addition to @tchellomello 's comment, could you update:

  * `deploy/olm-catalog/awx-operator/manifests/awx.ansible.com_awxs_crd.yaml`
  * `deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml` (Mark them as both `advanced` and `hiddden` - just like: https://github.com/ansible/awx-operator/blob/devel/deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml#L294-L298)
  
 Then looks good to me for inclusion

File: roles/installer/defaults/main.yml (hunk 1 above)
Comment (athak) [reply roles/installer/defaults/main.yml:56]: Added in 2a3f700.

**Comment by athak** [comment]
@Spredzy addressed in 2a3f700.

**Review by tchellomello — APPROVED** [review]
LGTM! Thanks @athak

**Comment by shanemcd** [comment]
Going to close and reopen to trigger the new GitHub Actions check.

**Pull request merged**